datumaro/plugins/data_formats/datumaro/exporter.py

```diff
@@ -34,6 +34,7 @@
 from datumaro.components.exporter import Exporter
 from datumaro.components.media import Image, MediaElement, PointCloud
 from datumaro.util import cast, dump_json_file
+from datumaro.version import VERSION
```

Comment:
Datumaro "library" version and Datumaro "format" version should be independent. This is because there would be no change for Datumaro "format" even if Datumaro "library" version goes up. For example, the current version of CVAT is 2.2, but the CVAT image dataset format version is 1.1 (https://opencv.github.io/cvat/v2.3.0/docs/manual/advanced/xml_format/).

Reply:
Good point!

I placed DATUMARO_FORMAT_VERSION at https://github.com/openvinotoolkit/datumaro/pull/842/files#diff-07058dec3606d234ac036aa0116869b8aad42d7b9f5b3b85c820674fea958085R5